recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: behst
+  version: 0.4
+
+source:
+  fn: a2a25d973909.zip
+  url: https://bitbucket.org/hoffmanlab/behst/get/a2a25d973909.zip
+  sha256: cf4cc7179e6311f72d350c4fd408930b367b568f4ca77469474c4e5044ffd015
+
+build:
+  number: 0
+  skip: True # [not py27 or osx]
```

Comment:
I don't see any dependencies that are py27 only, is there another reason why you're restricting the version here?